vars/runGradleCheck.groovy

```diff
@@ -71,7 +71,13 @@ void call(Map args = [:]) {
 
                 echo "Start gradlecheck"
                 GRADLE_CHECK_STATUS=0
-                ./gradlew clean && ./gradlew check -Dtests.coverage=true --no-daemon --no-scan || GRADLE_CHECK_STATUS=1
+                
+                if (uname -s | grep -i NT); then
+                   echo "Start gradlecheck on windows"
+                   ./gradlew clean && ./gradlew check --max-workers 8 -Dtests.coverage=true --no-daemon --no-scan || GRADLE_CHECK_STATUS=1
```

Comment:
Can the max-workers change in future? Asking if it can be something that should be passed as an argument?

Reply:
yes that would be ideal to pass the entire command from https://github.com/opensearch-project/OpenSearch/blob/main/.github/workflows/gradle-check.yml. This change is against windows CI only that runs nightly & that config is not in the other repo, so we'll also want to move that before being able to pass as a param bc we don't want to impact non windows checks.

Reply:
Maybe time to redo gradle check workflow as well? :) (cc: @bbarani)

Looks like it is distributed in around 3 repos today which is 
Build repo: 
https://github.com/opensearch-project/opensearch-build/blob/main/scripts/gradle/gradle-check.sh
https://github.com/opensearch-project/opensearch-build/blob/main/jenkins/gradle/gradle-check.jenkinsfile

This repo and OpenSearch repo.

Adding @peterzhuamazon to this conversation as he is the original contributor.

Reply:
Thanks @gaiksaya I will work with @mch2 on this.